File: Assignment 11/Java/8.3/RoomDimension.java
class RoomDimension {
    private double length;
    private double width;

    // Constructor
    public RoomDimension(double length, double width) {
        this.length = length;
        this.width = width;
    }

    public double getArea() {
        return length * width;
    }
}

File: Assignment 12/Tony Gradis C++/14.13/Code/RoomDimension.cpp
#include "RoomDimension.h"
#include <iostream>
using namespace std;

RoomDimension::RoomDimension(FeetInches l, FeetInches w) : length(l), width(w) {}

// Trả về diện tích phòng dưới dạng FeetInches
FeetInches RoomDimension::getArea() {
    return length.multiply(width);
}

// Hiển thị kích thước phòng
void RoomDimension::display() {
    cout << "Room Dimensions: ";
    length.display();
    cout << " x ";
    width.display();
    cout << endl;
}
